apisix/discovery/consul/init.lua

```diff
@@ -44,9 +52,15 @@ local events_list
 local consul_services
 
 local default_skip_services = {"consul"}
+local default_random_seed = 5
```

Comment:
Would a new variable name be better?
It confuses me as a `seed` as it don't do the seed stuff, would be better to like `range` or something else ?

Reply:
done